akka-persistence/src/main/mima-filters/2.5.25.backwards.excludes

```diff
@@ -0,0 +1,3 @@
+# #27411 changes not caught by previous mima versions
+## TimerMode parameter
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.persistence.fsm.PersistentFSM#Timer.unapply")
```

Comment:
Is that a real incompatibility?

Reply:
well, yes (https://github.com/akka/akka/pull/27034/files#diff-e8473b0da7c6cc5da8bfbbfa11043b3bR119), but that class is internal API, so that's OK. It's surprising it wasn't caught before, though.

Reply:
I see, thanks.